config/default/highlight_js.settings.yml

```
@@ -0,0 +1,12 @@
+copy_enable: true
+copy_bg_transparent: false
+copy_bg_color: '#4243b1'
+copy_txt_color: '#ffffff'
+copy_btn_text: ''
+copy_success_text: ''
+success_txt_color: '#ffffff'
+role_copy_access: {  }
```

YAMLlint (1.38.0)

[error] too many spaces inside empty braces

(braces)